shared/constants/wallets.js

```diff
@@ -257,7 +257,7 @@ const paymentToCounterpartyType = (p: Types.Payment): Types.CounterpartyType =>
   return 'stellarPublicKey'
 }
 
-const paymentToYourRole = (p: Types.Payment, username: string): 'sender' | 'receiver' => {
+const paymentToYourRole = (p: Types.Payment): 'sender' | 'receiver' => {
```

Comment:
not sure is paymentToCounterpartyType is entirely correct -- if source === target, then we won't have the name of the other account to display as the counterparty type. Will play around with it more...